website/Makefile

```diff
@@ -19,6 +19,8 @@ build:
 	cat ../_redirects | grep 'docs/' | awk '{ print "/v0.11" $$1 "\t" $$2 "\t" $$3 }' >> build/_redirects
 	cat ../_redirects | grep 'docs/' | awk '{ print "/v0.10" $$1 "\t" $$2 "\t" $$3 }' >> build/_redirects
 
+	cat ../_headers > build/_headers
```

Comment:
Just for the sake of my own curiosity, why did we need to do this? Is it just outputting the _headers file to the directory Netlify expects it to be in?

Reply:
After updating the headers file and not seeing changes, I noticed on the Netlify deploy page it said "no headers processed". The headers file was not being included in the final build. This line adds the headers file to the build folder, similar to how jekyll adds it to the _site folder.